test/cmdlineTests/recovery_ast_constructor/output

```diff
@@ -34,6 +34,7 @@ JSON AST (compact format):
       "canonicalName": "Error1",
       "contractDependencies": [],
       "contractKind": "contract",
+      "emittedEvents": [],
```

Comment:
Maybe I did not understand the semantic correctly. But I would have expected here one event for `emit L.e1(5);`

Reply:
Ah ok! Looks like that `emittedEvents` is just not yet implemented. I was not able to find any test that defines any `emittedEvents`. So I guess this PR is still in progress.